public/app/features/alerting/unified/components/import-to-gma/ConfirmConvertModal.tsx

```diff
@@ -292,14 +311,20 @@ This function is used to check if the rule should be skipped.
 It checks if the rule has the '__grafana_origin' label, and if the rule is from synthetics.
 If the rule has the '__grafana_origin' label, it is skipped.
 If the rule is from synthetics, it is skipped.
+If the rule is from integrations, it is skipped.
 */
 function shouldSkipRule(rule: RulerRuleDTO): boolean {
   // check if the rule has the '__grafana_origin' label
   const hasGrafanaOriginLabel = isPluginProvidedRule(rule);
   if (hasGrafanaOriginLabel) {
     return true;
   }
-  // check if the rule is from synthetics
+  // check if the rule is from intergrations by checking if the namespace starts with 'integrations-'
+  const isIntegration = rule.labels?.namespace?.startsWith('integrations-');
```

Comment:
nit: prefer a separate function here to detect rules that are "managed" so we can add more logic to it in the future :)